tasks/docker_nvidia_runtime.yml

```diff
@@ -1,15 +1,22 @@
 ---
-- name: Add Nvidia Docker GPG Key
-  ansible.builtin.shell: 'curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey | apt-key add -'
+- name: Get Nvidia signing key
   become: true
-  changed_when: true
+  ansible.builtin.get_url:
+    url: https://nvidia.github.io/nvidia-docker/gpgkey
+    dest: /etc/apt/trusted.gpg.d/nvidia.asc
```

Comment:
looking a bit more into it, it seems like this key needs to be handled a bit differently, something like the docker gpg key [here](https://github.com/robert-sandor/ansible-linux/blob/ec350249d123b8d69567ec957c73b408a88b645d/roles/docker/tasks/install_docker_debian.yml#L26)

either way, this need to be tested - not sure if you need an nvidia gpu to test

Reply:
agreed